saleor/checkout/utils.py

```diff
@@ -134,23 +134,6 @@ def add_variants_to_checkout(
     Otherwise, quantity will be added or replaced (if replace argument is True).
     """
 
-    # check quantities
-    country_code = checkout.get_country()
-    if not skip_stock_check:
```

Comment:
`skip_stock_check` becomes unnecessary after this change and could be removed from the signature.